Content.Server/Climbing/Components/ClimbingComponent.cs

```diff
@@ -1,92 +1,43 @@
 
-namespace Content.Server.Climbing.Components
+namespace Content.Server.Climbing.Components;
+
+[RegisterComponent]
+[ComponentReference(typeof(SharedClimbingComponent))]
+public sealed class ClimbingComponent : SharedClimbingComponent
 {
-    [RegisterComponent]
-    [ComponentReference(typeof(SharedClimbingComponent))]
-    public sealed class ClimbingComponent : SharedClimbingComponent
+    [ViewVariables(VVAccess.ReadWrite)]
+    public override bool IsClimbing
     {
-        [Dependency] private readonly IGameTiming _gameTiming = default!;
-        [Dependency] private readonly IEntityManager _entityManager = default!;
-
-        public override bool IsClimbing
-        {
-            get => base.IsClimbing;
-            set
-            {
-                if (base.IsClimbing == value)
-                    return;
-
-                base.IsClimbing = value;
-
-                if (value)
-                {
-                    StartClimbTime = IoCManager.Resolve<IGameTiming>().CurTime;
-                    EntitySystem.Get<ClimbSystem>().AddActiveClimber(this);
-                    OwnerIsTransitioning = true;
-                }
-                else
-                {
-                    EntitySystem.Get<ClimbSystem>().RemoveActiveClimber(this);
-                    OwnerIsTransitioning = false;
-                }
-
-                Dirty();
-            }
-        }
-
-        public override bool OwnerIsTransitioning
+        get => base.IsClimbing;
+        set
         {
-            get => base.OwnerIsTransitioning;
-            set
-            {
-                if (value == base.OwnerIsTransitioning) return;
-                base.OwnerIsTransitioning = value;
-                Dirty();
-            }
+            if (base.IsClimbing == value) return;
+            base.IsClimbing = value;
+            Dirty();
         }
+    }
 
-        /// <summary>
-        /// Make the owner climb from one point to another
-        /// </summary>
-        public void TryMoveTo(Vector2 from, Vector2 to)
+    public override bool OwnerIsTransitioning
+    {
+        get => base.OwnerIsTransitioning;
+        set
         {
-            if (!_entityManager.TryGetComponent<PhysicsComponent>(Owner, out var physicsComponent)) return;
-
-            var velocity = (to - from).Length;
-
-            if (velocity <= 0.0f) return;
-
-            // Since there are bodies with different masses:
-            // mass * 5 seems enough to move entity
-            // instead of launching cats like rockets against the walls with constant impulse value.
-            physicsComponent.ApplyLinearImpulse((to - from).Normalized * velocity * physicsComponent.Mass * 5);
-            OwnerIsTransitioning = true;
-
-            EntitySystem.Get<ClimbSystem>().UnsetTransitionBoolAfterBufferTime(Owner, this);
+            if (value == base.OwnerIsTransitioning) return;
+            base.OwnerIsTransitioning = value;
+            Dirty();
         }
+    }
```

Comment:
i forgot if we had a meeting topic over whether this was okay but i can't imagine these would be getting VVed ever, so they should probably just be done in the system and not as a property

Reply:
If we ever get map saving, then these'll probably need to be saved as part of that since they tell the system that the fixtures need to be changed back.